packages/dev-server-storybook/package.json

```diff
@@ -59,7 +59,7 @@
     "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@rollup/plugin-terser": "^0.4.4",
-    "@storybook/csf-tools": "^6.4.9",
```

Comment:
dev-server-storybook is an older pre-builder era package, we don't maintain it actively anymore and no need to update it, actually it should remain on 6.x

Reply:
noted